docs/qsimcirq_tutorial.ipynb

```diff
@@ -0,0 +1,594 @@
+        "# Define a circuit with intermediate measurements.\n",
+        "q0 = cirq.LineQubit(0)\n",
+        "circuit = cirq.Circuit(\n",
+        "    cirq.X(q0)**0.5, cirq.measure(q0, key='m0'),\n",
+        "    cirq.X(q0)**0.5, cirq.measure(q0, key='m1'),\n",
+        "    cirq.X(q0)**0.5, cirq.measure(q0, key='m2'),\n",
+        ")\n",
+        "\n",
+        "# Simulate the circuit with qsim and return just the measurement values.\n",
+        "print('qsim results:')\n",
+        "qsim_simulator = qsimcirq.QSimSimulator()\n",
+        "qsim_results = qsim_simulator.run(circuit, repetitions=5)\n",
+        "print(qsim_results)"
+      ],
+      "execution_count": 7,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "qsim results:\n",
+            "m0=00001\n",
+            "m1=11100\n",
+            "m2=00001\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KhPHBKpi0lYE",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Amplitude evaluation\n",
+        "\n",
+        "qsim can also calculate amplitudes for specific output bitstrings."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "aCbvXO6M1jWB",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 104
+        },
+        "outputId": "e2d3a80f-00c1-4248-e7ef-175e007eadf1"
+      },
+      "source": [
+        "# Define a simple circuit.\n",
+        "q0, q1 = cirq.LineQubit.range(2)\n",
+        "circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))\n",
+        "\n",
+        "# Simulate the circuit with qsim and return the amplitudes for |00) and |01).\n",
+        "print('Cirq results:')\n",
+        "cirq_simulator = cirq.Simulator()\n",
+        "cirq_results = cirq_simulator.compute_amplitudes(\n",
+        "    circuit, bitstrings=[0b00, 0b01])\n",
+        "print(cirq_results)\n",
+        "print()\n",
+        "\n",
+        "# Simulate the circuit with qsim and return the amplitudes for |00) and |01).\n",
+        "print('qsim results:')\n",
+        "qsim_simulator = qsimcirq.QSimSimulator()\n",
+        "qsim_results = qsim_simulator.compute_amplitudes(\n",
+        "    circuit, bitstrings=[0b00, 0b01])\n",
+        "print(qsim_results)"
+      ],
+      "execution_count": 8,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cirq results:\n",
+            "[0.70710677+0.j 0.        +0.j]\n",
+            "\n",
+            "qsim results:\n",
+            "[(0.7071067690849304+0j), 0j]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zy5XRdkm219l",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Performance benchmark\n",
```

Comment:
Perhaps it makes sense to mention that qsimcirq can use many threads for better performance.

Reply:
Done. Also added a short example, although the effect may be limited by the cores of the Jupyter instance.